stdlib/public/RemoteInspection/TypeLowering.cpp

```diff
@@ -1960,6 +1960,7 @@ class EnumTypeInfoBuilder {
     unsigned NonPayloadCases = 0; // `case a`
     unsigned NonGenericEmptyPayloadCases = 0; // `case a(Void)` or `case b(Never)`
     unsigned NonGenericNonEmptyPayloadCases = 0; // `case a(Int)` or `case d([Int?])`
+    unsigned IndirectPayloadCases = 0; // `indirect case a(T)` or `indirect case a(Int)`
```

Comment:
Can you add a quick comment that indirect takes precedence over generic, and why? Might save our future selves a bit of confusion.

Reply:
The new version includes comments.  Thanks for the poke.

Reply:
Looks great!